core/src/main/java/io/grpc/transport/HttpUtil.java

```diff
@@ -69,6 +69,11 @@
   public static final String TE_TRAILERS = "trailers";
 
   /**
+   * The Timeout header name.
+   */
+  public static final String TIMEOUT = "grpc-timeout";
```

Comment:
it's used to construct `TIMEOUT_KEY`; leaving it here rather than a bare string literal felt better.